internal/persiststorage/persiststorage_configmap.go

```diff
@@ -15,8 +15,8 @@ import (
 )
 
 type KubeConfigMapStore struct {
-	Name       string
```

Comment:
I think it wasn't introduced in this patch, but I just noticed that this type isn't using the builder pattern like in all other types of the project. Please take note and fix it (in a different pull request).

Reply:
OK. Will do that.